include/rocksdb/table.h

```diff
@@ -47,6 +47,7 @@ enum ChecksumType : char {
   kNoChecksum = 0x0,
   kCRC32c = 0x1,
   kxxHash = 0x2,
+  kxxHash64 = 0x3,
```

Comment:
Forgot to mention one thing. We should add a comment here, saying it's the lower 32 bits of xxhash64, not xxhash64.